geomapshark/views.py

```diff
@@ -68,28 +106,39 @@ def permit_author_create(request):
 
     if is_valid:
         new_user = djangouserform.save()
+        # email wasn't verified yet, so account isn't active just yet
+        new_user.is_active = False
```

Comment:
Since we are setting accounts as inactive by default, we should probably have a different error message when the user tries to login before activating the account. Currently "Votre mot de passe et votre nom d'utilisateur ne correspondent pas" is kind of misleading